emperor/support_files/js/view.js

```diff
@@ -414,5 +435,70 @@ DecompositionView.prototype.setCategory = function(attributes,
   return dataView;
 };
 
+/**
+ *
+ * Hide edges where plottables are present.
+ *
+ * @param {Plottable[]} plottables An array of plottables for which the edges
+ * should be hidden. If this object is not supplied, all the edges are hidden.
+ */
+DecompositionView.prototype.hideEdgesForPlottables = function(plottables) {
+  // no edges to hide
+  if (this.decomp.edges.length === 0) {
+    return;
+  }
+
+  var u, v, j = 0, positionsLeft, positionsRight, hideAll;
+
+  positionsLeft = this.lines.left.geometry.attributes.position.array;
+  positionsRight = this.lines.right.geometry.attributes.position.array;
+
+  hideAll = plottables === undefined;
+
+  this.decomp.edges.forEach(function(edge) {
+    u = edge[0];
+    v = edge[1];
+
+    if (hideAll ||
+        (plottables.indexOf(u) !== -1 || plottables.indexOf(v) !== -1)) {
+      positionsLeft[(j * 6)] = 0;
```

Comment:
The 1->5 sounds/looks like a variable or something like that ... or why 5?

Reply:
Thus, this is a global parameter defined by BufferGeometry and if changed this will change, right? Therefore, should we have it encapsulated somehow (method?) so we don't need to change multiple places if we move to something different? If the answer is yes and the changes are complex, opening an issue is fine.

Reply:
Fair enough, I've created a wrapper method for these changes. 👍